packages/data-store/src/types/contact/contact.ts

```diff
@@ -214,24 +209,29 @@ export type ElectronicAddressType = 'email' | 'phone'
 
 export type PhysicalAddressType = 'home' | 'visit' | 'postal'
 
-export enum IdentityRoleEnum {
+export enum IdentityRole {
   ISSUER = 'issuer',
   VERIFIER = 'verifier',
   HOLDER = 'holder',
 }
 
-export enum ConnectionTypeEnum {
+export enum ConnectionType {
   OPENID_CONNECT = 'OIDC',
   SIOPv2 = 'SIOPv2',
   SIOPv2_OpenID4VP = 'SIOPv2+OpenID4VP',
 }
 
-export enum CorrelationIdentifierEnum {
+export enum CorrelationIdentifierType {
   DID = 'did',
   URL = 'url',
 }
 
-export enum PartyTypeEnum {
+export enum PartyTypeType {
   NATURAL_PERSON = 'naturalPerson',
   ORGANIZATION = 'organization',
 }
+
+export enum PartyOrigin {
+  INTERNAL = 'internal',
```

Comment:
Please make enum names and values the same in case. We have problems with these when they are used in DTOs/REST APIs

Reply:
as discussed, i made this one the same, name and value are uppercase now